doc/user-manual/contribute/documentation.rst

```diff
@@ -33,7 +33,7 @@ Rendering documentation locally
 
   To see the list of available targets, execute ``make help``
   in ``doc/user-manual``. E.g., call ``make html`` to build the
-  documentation in html format.
+  documentation in the html format.
```

Comment:
Not sure about this definitive article here. Confer:
a) "Dance this song in Gangnam style!"  
b) "Dance this song in the Gangnam style!"

My gut prefers a).

Reply:
I'm not a native English speaker, but to me it does sound better with the article. Nevertheless, I might be wrong. I'll revert this change.

Reply:
I just force-pushed a commit without the "the html format" change.